.github/scripts/lib/goal-run-stage-core.ps1

```
@@ -269,7 +314,22 @@ function Get-GoalRunIssueComments {
+        # ambient-cwd paths below, which is wrong whenever -RepoRoot
+        # differs from the process cwd.
+        try {
+            $remoteUrl = & $GitCliPath -C $RepoRoot remote get-url origin 2>$null
+            if ($LASTEXITCODE -eq 0 -and -not [string]::IsNullOrWhiteSpace($remoteUrl)) {
+                $match = [regex]::Match($remoteUrl, 'github\.com[:/](.+?)(?:\.git)?/?$')
+                if ($match.Success) { $ownerRepo = $match.Groups[1].Value.Trim() }
+            }
+        }
+        catch {
+            # Falls through to the ambient-cwd resolution below.
+        }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)